tap_square/discover.py

```diff
@@ -8,7 +8,7 @@ def get_abs_path(path):
     return os.path.join(os.path.dirname(os.path.realpath(__file__)), path)
 
 # NB: These streams cannot be queried using Sandbox OAuth credentials
-PRODUCTION_ONLY_STREAMS = {'roles', 'bank_accounts', 'settlements'}
+PRODUCTION_ONLY_STREAMS = {'roles', 'bank_accounts'}
```

Comment:
should payouts be added here as well or are we able to query the new object from sandbox?

Reply:
I tried querying the payouts using sandbox creds where I'm getting the empty response {}. So, adding the payouts in prod only streams.